python/src/scipp/plot/widgets.py

```diff
@@ -156,7 +158,7 @@ def _add_masks_controls(self, masks):
                 for key in masks[name]["names"]:
                     self.mask_checkboxes[name][key] = ipw.Checkbox(
                         value=masks[name]["names"][key],
-                        description="{}:{}".format(name, key),
+                        description="{}:{}".format(escape(name), escape(key)),
```

Comment:
I'm not sure how you decide what needs escaping and what doesn't, but there are the dimension buttons (around L104) that have the dim labels as description, maybe this also needs escaping?

Then I guess you may also need to look at `panel1d.py`.

Reply:
I decided by trying out all sorts of outputs.

Are you referring to this? https://github.com/scipp/scipp/blob/2290ddeda0d5388a4261f08cf50f969250d306c7/python/src/scipp/plot/widgets.py#L106

It looks like the widgets escape the string for us:
![plot_buttons](https://user-images.githubusercontent.com/11393224/108040749-f9ba8a80-703d-11eb-860d-7bae8d463392.png)

The figures all seem to work properly, too. At least I didn't find an example where it broke.

Reply:
Yes this is what i was talking about.
I tried a few more things, but I think what you got it all (weird that the checkboxes descriptions need special handling)

Reply:
Well, they are formatted by ipywidgets. Our problems were all with places where we constructed HTML manually.